### PR TITLE
ARROW-13056: [MATLAB] Add a matlab label for dev Pull Requests

### PR DESCRIPTION
**Pull request by tahsinH**
Create a "lang-matlab" label for Pull Requests

https://issues.apache.org/jira/browse/ARROW-13056
Email : thread 
https://lists.apache.org/thread.html/rb348bb8b3c998ba081b68fa24ab86ceb542ecc938bced92b95516cc3%40%3Cdev.arrow.apache.org%3E

### CODE REVIEW

File: .github/workflows/dev_pr/labeler.yml [hunk 1]
```diff
@@ -45,6 +45,9 @@ lang-R:
 lang-ruby:
   - ruby/**/*
 
+lang-matlab:
```
Comment (kou) [inline .github/workflows/dev_pr/labeler.yml:48]: Could you put this between `lang-julia` and `lang-python`?
This list is sorted in alphabetically.

**Comment by tahsinH** [comment]
Seems like the travis CI failure is unrelated to my change. I am only changing a barebone yaml file, that shouldn't effect C++ builds.  I am seeing similar symptoms in other PRs. e.g. , I see _the same error_ in the logs 

**FAILED: src/arrow/CMakeFiles/arrow_objlib.dir/Unity/unity_20_cxx.cxx.o**
for the ubuntu build in other PR build failure logs...

https://travis-ci.com/github/apache/arrow/jobs/520441357
https://travis-ci.com/github/apache/arrow/jobs/517929691

**Comment by tahsinH** [comment]
@kou could you help in approving the workflow (since I am first time contributor, one of the maintainers need to approve some workflow) and/or review the changes. Thanks

**Review by kou — APPROVED** [review]
+1

Sorry. I missed this...

**Pull request closed without merging**